### DISCUSSION TITLE
Project template question

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#174] [Q&A] [answered]
Once I have the template set up, should I edit and add my code to the README.md file, or should I create a new file? Does it matter? I thought I read somewhere about some code being pre-set for something, but I don't see that pre-set code anywhere in the template. Did I miss something?

**Comment by CJ3315** [comment, marked as answer]
I believe that you can do either/or, just as long as the code can be committed and you can see updates on your repository? @kensnyder

**Reply by kensnyder** [reply]
I don't know exactly what Sean had in mind, but as long as I can find your work, it doesn't matter to me where you put it. As @derekscholes mentions below, the README won't actually show up as part of the site. So if the README is with the site or separate, it won't affect how your site looks.

**Comment by derekscholes** [comment]
So the readme file doesn't create the website. You would need to create separate files with your code. Take a look at my page and let me know if you have other questions.

https://github.com/derekscholes/derekscholes.github.io

Feel free to go into my codespace as well.
